docs/lineage/dagster.md

```diff
@@ -0,0 +1,105 @@
+# Dagster Integration
+
+DataHub supports integration of
+
+- Dagster Pipeline metadata
+- Job and Op run information as well as
+- Lineage information when present
+
+## Using Datahub's Dagster Sensor
+
+Dagster sensors allow us to perform some action based on some state change. Datahub's defined dagster sensor will emits metadata after every dagster pipeline run execution. This sensor is able to register both pipeline success as well as failures. For more details about Dagster sensors please refer [Sensors](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors).
```

Comment:
Change `This sensor is able to register both pipeline success as well as failures` to `This sensor is able to emit both pipeline success as well as failures`